.github/renovate.json

```diff
@@ -9,6 +9,6 @@
   "dependencyDashboardAutoclose": true,
   "enabledManagers": ["terraform"],
   "terraform": {
-    "ignorePaths": ["**/context.tf"]
+    "ignorePaths": ["**/context.tf", "**/examples/version2/**"]
```

Comment:
seems not to be adding anything, as there is no such folder I can spot.

Reply:
Right, the directory go renamed.